test/unit/modunit_pico_mld.c

```diff
@@ -118,16 +119,12 @@ END_TEST
 START_TEST(tc_pico_mld_report_expired)
 {
     struct mld_timer *t = PICO_ZALLOC(sizeof(struct mld_timer));
-    struct pico_ip6 zero = {{0}};
-    struct mcast_parameters p;
+    struct pico_ip6 zero = {0};
```

Comment:
and other compilers might also complain about it.
I'm still tending towards using `{0}`, because it's the standard.

Older versions of GCC can be silenced by passing `-Wno-missing-brace` (just for the unit tests, please). I still don't quite understand why the same compiler would complain about these things in the test files, and let them pass in the others (the flags seem to be the same), but since it's the wrong behaviour from the compiler anyhow, let's just silence him here.

Unless somebody has a better idea :-)